sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -255,6 +258,7 @@ internal class BitmapWidgetRenderer(
     this.halfBitmapHeight = bitmap.height / 2f
     updateTranslateMatrix()
     updateVertexBuffer()
+    updateBitmap = true
```

Comment:
just a question. should you also set this flag inside `setRotation` function?

Reply:
No, it should only be set when the source Bitmap itself is changed